react/features/prejoin/components/Prejoin.native.tsx

```diff
@@ -92,31 +84,54 @@ const Prejoin: ({ navigation }: Props) => JSX.Element = ({ navigation }: Props)
         );
     }, []);
 
+    const joinButtonStyles = displayName
+        ? styles.primaryButton : styles.primaryButtonDisabled;
```

Comment:
We only want to show the button as disabled if it's actually disabled, check with `joinButtonDisabled` instead.